src/pages/shoppingListsPage/shoppingListsPage.test.tsx

```diff
@@ -273,6 +278,246 @@ describe('ShoppingListsPage', () => {
+        const wrapper = renderAuthenticated(
+          <PageProvider>
+            <GamesProvider>
+              <ShoppingListsProvider>
+                <ShoppingListsPage />
+              </ShoppingListsProvider>
+            </GamesProvider>
+          </PageProvider>,
+          'http://localhost:5173/shopping_lists'
+        )
+
+        await waitFor(() => {
+          expect(wrapper.queryByTestId('pulseLoader')).toBeFalsy()
+          expect(
+            wrapper.getByText(
+              /You need a game to use the shopping lists feature\./
+            )
+          ).toBeTruthy()
+          expect(wrapper.getByText('Create a game')).toBeTruthy()
+        })
+      })
+
+      test('shows a message that you need a game', async () => {
+        const wrapper = renderAuthenticated(
+          <PageProvider>
+            <GamesProvider>
+              <ShoppingListsProvider>
+                <ShoppingListsPage />
+              </ShoppingListsProvider>
+            </GamesProvider>
+          </PageProvider>,
+          'http://localhost:5173/shopping_lists'
+        )
+
+        await waitFor(() => {
+          expect(
+            wrapper.getByText(
+              /You need a game to use the shopping lists feature\./
+            )
+          ).toBeTruthy()
+          expect(wrapper.getByText('Create a game')).toBeTruthy()
+        })
+      })
+
+      test('matches snapshot', async () => {
+        const wrapper = renderAuthenticated(
+          <PageProvider>
+            <GamesProvider>
+              <ShoppingListsProvider>
+                <ShoppingListsPage />
+              </ShoppingListsProvider>
+            </GamesProvider>
+          </PageProvider>,
+          'http://localhost:5173/shopping_lists'
+        )
+
+        await wrapper.findByText(
+          /You need a game to use the shopping lists feature\./
+        )
+
+        expect(wrapper).toMatchSnapshot()
+      })
+    })
+  })
+
+  describe('creating a game from the shopping lists page', () => {
+    describe('when successful', () => {
+      const mockServer = setupServer(
+        getGamesEmptySuccess,
+        postGamesSuccess,
+        getShoppingListsEmptySuccess
+      )
+
+      beforeAll(() => mockServer.listen())
+      beforeEach(() => mockServer.resetHandlers())
+      afterAll(() => mockServer.close())
+
+      test('displays the form', async () => {
+        const wrapper = renderAuthenticated(
+          <PageProvider>
+            <GamesProvider>
+              <ShoppingListsProvider>
+                <ShoppingListsPage />
+              </ShoppingListsProvider>
+            </GamesProvider>
+          </PageProvider>,
+          'http://localhost:5173/shopping_lists'
+        )
+
+        const link = await wrapper.findByText('Create a game')
+
+        act(() => fireEvent.click(link))
+
+        expect(wrapper.getByTestId('gameForm')).toBeTruthy()
+        expect(wrapper.getAllByText('Create Game')).toBeTruthy()
```

Comment:
This will always be truthy. Need to call `.length` for the possibility that it is 0.